spec/fuzzy-finder-spec.coffee

```diff
@@ -585,6 +585,20 @@ describe 'FuzzyFinder', ->
           expect(atom.workspace.panelForItem(projectView).isVisible()).toBe false
           expect(inputView).toHaveFocus()
 
+  describe "when the filter text contains an absolute path", ->
+    it "it matches the given path", ->
```

Comment:
He's suggesting replace:

```coffee
it "it matches the given path", ->
```

with

```coffee
it "matches the given path", -"
```

You don't need to use the word `it` twice in the same line :grinning: